tsconfig.json

```diff
@@ -13,6 +13,6 @@
     "declaration": true,
     "resolveJsonModule": true
   },
-  "include": ["src/", "script/", "docs/", "config/types/"],
+  "include": ["src/", "script/", "docs/", "config/types/", "config/**/*.test.ts", "test-utilities/"],
```

Comment:
could test-utilities go under src instead? This `includes` array is getting quite large and might make it less predictable for others when it comes to debugging.

Reply:
Yes, does that make sense? I think we wanted to [move tokens into source (`src/`)](https://github.com/github/primer/pull/1309/files?short_path=722b130#diff-722b130da8c969a084abdcb34977e310c071d1af7f0458ef6f8a35a7155c1a4c) so would this still make sense then? And if yes would we move `script/` into source as well? And `config`?

Is there a common way to structure this at github?

Reply:
not an enforced convention AFAIK, but you typically not want to overload the includes array and rather use it as a catch-all. Also you have config/types in there. Not sure what that is exactly, but just on naming it may be worth putting that under the [typeRoots](https://www.typescriptlang.org/tsconfig#typeRoots) property instead?